libs/scripts/scripts.js

```diff
@@ -41,6 +41,12 @@ const stageDomainsMap = {
     '^https://business.adobe.com/blog': 'https://main--bacom-blog--adobecom.hlx.page',
     '^https://www.adobe.com': 'origin',
   },
+  '^https://.*--milo--.*.aem.page': {
+    '^https://www.adobe.com/acrobat': 'https://main--dc--adobecom.hlx.page',
+    '^https://business.adobe.com(?!/blog)': 'https://business.stage.adobe.com',
+    '^https://business.adobe.com/blog': 'https://main--bacom-blog--adobecom.hlx.page',
+    '^https://www.adobe.com': 'origin',
+  },
```

Comment:
This might be tricky to keep track of, we're now redirecting .aem. traffic to .hlx. on consumers, sincer they're not on Helix 5 just yet, but we'll need to update these at one point. Shouldn't be an issue at the moment, though.

Reply:
Yeah, we'll need to live with the mix for now 😢

Reply:
Agree, we'll need to update at one point, but we can go live with this for now.